### ISSUE TITLE
Wrong background color in light mode

### ISSUE THREAD

**Issue by gunnarbittersmann** [cssbattle/forum#251]
In light mode on https://cssbattle.dev/me, the table head has low color contrast with black on dark gray.
<img width="1324" alt="screenshot" src="https://user-images.githubusercontent.com/409844/163730356-c57e698c-7973-4478-9d0e-3a3a282a74f1.png">

**Comment by chinchang** [comment]
fixed!
